### ISSUE TITLE
Possible bug in mnist_illustrative_example

### ISSUE THREAD

**Issue by sidml** [anilkagak2/PDE_GlobalLayer#1]
Hi.
I was going through the code and noticed that the CNN model has 522 parameters instead of 524 parameters.
I think this is because the final batch norm and activation layers are missing in the CNN model.
Residual and PDE model have these layers. 
For example: In [PDE Model](https://github.com/anilkagak2/PDE_GlobalLayer/blob/main/train_mnist_illustrative_example.py#L108), the bn & activation are present here.

I re-implemented the CNN model after adding batchnorm & activation layers and observed that the accuracy of CNN model and PDE model is similar (PDE model is probably 0.5% more accurate).

The results seem to be dependent on the seed in my implementation. Hence I am curious if you used the same seed used for evaluating all three models ?

Thanks for you time & my apologies if I am misunderstanding something.
